LICENSE

```diff
@@ -1,7 +1,7 @@
 MIT License
 
 Copyright (c) 2020 sa
-Copyright (c) 2020-2021 MeiliSearch
+Copyright (c) 2020-2021 Meilisearch
```

Comment:
I think the line above should be removed

Reply:
Good catch!